app/src/main/java/com/aplication/carsales/main_module/view_model/MainViewModel.kt

```diff
@@ -1,10 +1,12 @@
 package com.aplication.carsales.main_module.view_model
 
+import android.util.Log
```

Comment:
If you're not using this import can you please delete it

Reply:
fix it!